src/App.js

```diff
@@ -1,12 +1,16 @@
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
 import Header from "./components/Header.js";
 
 function App() {
   return (
     <div className="grid grid-cols-4 gap-4 p-7">
-      <Header className="rounded-lg	flex bg-yellow-500" name="To Do"></Header>
-      <Header className="rounded-lg	flex bg-green-500" name="Doing"></Header>
-      <Header className="rounded-lg	flex bg-yellow-400" name="Done"></Header>
-      <Header className="rounded-lg	flex bg-blue-500" name="Approve"></Header>
+      <DndProvider backend={HTML5Backend}>
+        <Header className="rounded-lg	flex bg-yellow-500" name="To Do"></Header>
+        <Header className="rounded-lg	flex bg-green-500" name="Doing"></Header>
+        <Header className="rounded-lg	flex bg-yellow-400" name="Done"></Header>
+        <Header className="rounded-lg	flex bg-blue-500" name="Approve"></Header>
+      </DndProvider>
```

Comment:
If there is not inner html, the JSX tags can be self-closing
ex: `<Header .... />`

Reply:
Cool. Thank you